src/actions/eventTypeActions.js

```diff
@@ -30,3 +35,23 @@ export const getEventTypeActionSet = (eventTypeId) => ({
         failureReason
     })
 })
+
+export const getEventTypesActionSet = () => ({
+    try: () => ({
+        type: TRY_GET_EVENT_TYPES
+    }),
+
+    succeed: (eventTypes) => (dispatch) => {
```

Comment:
Don't need the extra layer of dispatch anymore, this can just be

(eventTypes) => ({
type: GET_EVENT_TYPES_SUCCESS,
eventTypes
})

Reply:
Fixed.

Reply:
Fixed.